src/__tests__/lib/containers/widgets/query-filter/QueryFilter.test.tsx

```diff
@@ -252,5 +233,22 @@ describe('QueryFilter tests', () => {
       expected.query = { ...expected.query, selectedFacets: expectedResult }
       expect(mockExecuteQueryRequest).toHaveBeenCalledWith(expected)
     })
+
+    it('renders all available facet chips', async () => {
+      init()
+      expect(await screen.findAllByRole('button')).toHaveLength(
+        mockQueryResponseData.facets.length,
+      )
+    })
+
+    it('facet chip changes color onClick', async () => {
+      init()
+      const facetChip = await screen.findByRole('button', {
+        name: mockQueryResponseData.facets[0].columnName,
+      })
+      expect(facetChip.className).toEqual('Chip Checked')
+      await userEvent.click(facetChip)
+      expect(facetChip.className).toEqual('Chip ')
+    })
   })
```

Comment:
@yjyoo773 can you add one test where `facetsToFilter` has 1-2 elements and verify that only those chips are shown?

After that I think this would be good to merge!

Reply:
^^^ it would be sufficient to just add that check on   `it('should respect facetsToFilter', ...`